src/main/java/com/netgrif/application/engine/resource/service/NetgrifResourceLoaderProcessor.java

```diff
@@ -0,0 +1,31 @@
+package com.netgrif.application.engine.resource.service;
+
+import com.netgrif.application.engine.resource.domain.NetgrifResourceLoader;
+import org.springframework.context.ResourceLoaderAware;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.ProtocolResolver;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Component;
+
+@Component
+public class NetgrifResourceLoaderProcessor implements ResourceLoaderAware, ProtocolResolver {
```

Comment:
Rename class to `ExternalResourceLoaderProcessor`

Reply:
done